benchmarks/README.md

```diff
@@ -143,3 +154,7 @@ We would like to add more data sets.  In general we will favor data sets
 that make the overall suite diverse without being too large or having
 too many similar tests.  Ideally everyone can run through the entire
 suite without the test run getting too long.
+
+## Sample result
+
+A Sample benchmark result can be seen [here](https://github.com/google/protobuf/blob/master/docs/Performance.md)
```

Comment:
Move this to the first paragraph. I.e.:

This directory contains ... test a variety of performance scenarios against your protobuf language runtime. **If you are looking for performance numbers of officially support languages, see [here]**.

Reply:
Done